.goreleaser.yml

```diff
@@ -0,0 +1,42 @@
+archives:
+  - name_template: '{{ .ProjectName }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}'
+    format: zip
+before:
+  hooks:
+    - go mod download
+builds:
+  - binary: '{{ .ProjectName }}_{{ .Version }}'
+    env:
+      - CGO_ENABLED=0
```

Comment:
Not sure if `go mod download` does anything that this may affect, but you might want to move it to the top-level `env` section, so that any other `go` commands outside of `builds` can pick it up too.